index.html

```diff
@@ -361,5 +361,36 @@ <h4>Skills</h4>
 
     </section>
 
+    <section class="contact">
+      <div class="contact-text">
+        <p>
+          I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.
+        </p>
+      </div>
+
+      <form action="" class="contact-form" method="post">
+        <input type="text" name="name" class="contact-form-text" maxlength="30" placeholder="Full name" required>
+        <input type="email" name="email" class="contact-form-email" placeholder="Email address" required>
+        <textarea name="contact" name="message" class="contact-textarea" maxlength="500" placeholder="Enter your text here" required></textarea>
+        <button type="submit" class="contact-button">Get in touch</button>
+      </form>
+    </section>
+
+    <footer>
+      <div class="footer-line"></div>
+      <div class="footer-socials">
+        <ul>
+          <li><a href="#"><img src="assets/Group.png" alt=""/></a></li>
+          <li><a href="#"><img src="assets/Vector.png" alt=""/></a></li>
+          <li><a href="#"><img src="assets/Vector2.png" alt=""/></a></li>
+          <li><a href="#"><img src="assets/Vector3.png" alt=""/></a></li>
+          <li><a href="#"><img src="assets/Vector4.png" alt=""/></a></li>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please change the color of your social network icons to require [Fgma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=mUwfyaAzix1prFLg-0) color (`#505F79`) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/205943454-b3c08515-fce3-4005-aad9-fc58a9406062.png)